server/paths/refer.ts

```diff
@@ -17,6 +17,7 @@ const confirmOasysPath = showReferralPath.path('oasys-confirmed')
 const checkAnswersPath = showReferralPath.path('check-answers')
 const completeReferralPath = showReferralPath.path('complete')
 const submitReferralPath = showReferralPath.path('submit')
+const reasonForReferralPath = showReferralPath.path('reason-for-referral')
```

Comment:
Interested to hear your thoughts on this, but I used `oasys-confirmed` for the `confirmOasysPath` to have it correspond with the property name (feels more RESTful to me), so we're looking at a referral `/referrals/:referralId` and from that its `/oasysConfirmed` property. Here I wonder if `/reason` would offer greater parity between entities and routes (and actions)?

Reply:
I was in 2 minds, I did go with `/reason` to begin with and then started to think ahead to if there was ever another "reason" for anything during the referral process but at the moment there isn't so I'm happy to go with just `/reason`